code/game/atoms.dm

```diff
@@ -826,6 +825,10 @@
 			add_overlay(new_overlays)
 		. |= UPDATE_OVERLAYS
 
+	if(updates & UPDATE_GREYSCALE)
```

Comment:
Shouldn't we check here that they do have a greyscale config? calling ``update_appearance`` will no args will otherwise call this.

Reply:
The check already happens in `update_greyscale`, I can insert one here too but the original one should stay to avoid errors

Reply:
I was just thinking of possible proc overhead, I dunno how many times this is called on average in a round